src/drivers/core.go

```diff
@@ -27,6 +25,21 @@ type CodeHostingDriver interface {
 	HostingServiceName() string
 }
 
+// config defines the configuration data needed by the driver package.
+type config interface {
```

Comment:
In order to remove the dependency on the `git` package, I have unified all driver-specific interfaces into this interface.